implementations/rust/ockam/ockam_command/src/node/create.rs

```diff
@@ -45,6 +45,7 @@ after_long_help = docs::after_help(AFTER_LONG_HELP)
 pub struct CreateCommand {
     /// Name of the node or a configuration to set up the node.
     /// The configuration can be either a path to a local file or a URL.
+    /// TODO: Use Option<String>
```

Comment:
What's the context of this?

Reply:
With current approach it's impossible to see if a user has passed "_default_node_name" or no value at all. Then later in the code you can see
```
if name == DEFAULT_NODE_NAME {
        name = random_name();
}
```
Which is now a hacky way to figure out if the user has provided no value. It would be a surprise for a user who provided "_default_node_name" and then ended up with a random node name, right? Not sure that's a realistic concern in this particular situation (due to low chance someone picking "_default_node_name" as the node name), but using Option would make it a clear distinction and as a consequence - easier to read the code.

Reply:
I was checking previous versions of the command and it's never been an `Option<String>`, although conceptually it's always been optional. The only explanation I can find is that it's easier to work with a `String`, given all the logic we have around this argument.

Reply:
I think if it's conceptually optional, it's better make it Optional, otherwise it can bite us in very unobvious way eventually :)